Tests/SpriteKitTests/SpriteKitTests.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  SpriteKitTests.swift
+//  SwifterSwift
+//
+//  Created by Olivia Brown on 5/28/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+import XCTest
+import SpriteKit
```

Comment:
Instead of importing `SpriteKit`, let's do `@testable import SwifterSwift`. It looks like CI can't find the method `descendants` from `SwifterSwift`.